src/Services/CourseRegistration/CourseRegistration.UnitTests/UnitTest1.cs

```diff
@@ -0,0 +1,14 @@
+using System;
```

Comment:
Please write unit test or remove the file

Reply:
Removed file. please check.